README.md

```diff
@@ -55,6 +55,13 @@ From here, you can start the jupyter notebook by using:
 
 where the parameters can be found at the top of the `Metrics Template.ipynb`
 
+## Steps to convert to PDF
+
+1. Follow the steps to run scripts and then enter into the `output.ipynb` and add tags to code cells that you don't want to appear in the final PDF. The name of `output.ipynb` will beome the title for the PDF. 
```

Comment:
Is this necessary after the changes you made?
It may be helpful to explain that the tags will be referenced in the pdf conversion call

Reply:
Will do. Thanks for catching this.